### PR TITLE
Fix noindex,nofollow robots meta tag

### PR DESCRIPTION
**Pull request by Danoosh**
Fix noindex,nofollow robots meta tag with category sorting

### CODE REVIEW

**Comment by Danoosh** [comment]
@eternoendless Please merge this pr!

**Comment by eternoendless** [comment]
Hi @Danoosh, could you please explain what this change does and how can it be tested? Thanks

**Comment by Danoosh** [comment]
Hi @eternoendless, This do same as #99 but keep follow,index in header of category pages.
Disable "Allow indexing robots to use category index" in configuration and see a category page in <head> find <meta name="robots" content="follow,index"> for test. without this change you must see "nofollow,noindex" in content property.

File: blocklayered.php [hunk 1]
```diff
@@ -1934,29 +1930,30 @@ public function getProductByFilters($selected_filters = array())
...
-
-			Db::getInstance()->execute('CREATE TEMPORARY TABLE '._DB_PREFIX_.'cat_filter_restriction ENGINE=MEMORY
-														SELECT cp.id_product, MIN(cp.position) position FROM '._DB_PREFIX_.'category_product cp
-														STRAIGHT_JOIN `'._DB_PREFIX_.'product` p ON (p.id_product=cp.id_product)
-														'.$price_filter_query_in.'
-														'.$query_filters_from.'
-														WHERE cp.`id_category` IN ('.implode(',', $categories).') '.$query_filters_where.'
-														GROUP BY cp.id_product ORDER BY position, id_product', false);
+			$home_category = Configuration::get('PS_HOME_CATEGORY');
+			$id_parent = (int)Tools::getValue('id_category', Tools::getValue('id_category_layered', $home_category));
```
Comment (eternoendless) [inline blocklayered.php]: It looks like these two have already been set in lines 1789 and 1791

**Review by eternoendless — COMMENTED** [review]
There's a small piece of duplicate code, otherwise it looks good to me.

**Comment by PierreRambaud** [comment]
Hi and thanks for your contribution.
As it says in the README, this repository is abandonned, try our new module https://github.com/PrestaShop/ps_facetedsearch.
Kind regards

**Pull request closed without merging**